services/ui-src/src/components/reports/DynamicDrawerReportPage.tsx

```diff
@@ -94,17 +90,15 @@ export const DynamicDrawerReportPage = ({
         </Box>
       )}
       <Button onClick={() => openRowDrawer("")} type="submit">
-        {dynamicTable.addEntityText}
+        Add TEMPORARY
```

Comment:
Just triple checking, this is intentional, right?

Reply:
lol yeah. good eye though.